frontend/src/scenes/pipeline/hogfunctions/filters/HogFunctionFiltersInternal.tsx

```diff
@@ -29,14 +33,18 @@ const setSimpleFilterValue = (value: string): HogFunctionFiltersType => {
     }
 }
 
-export function HogFunctionFiltersInternal(): JSX.Element {
+export function HogFunctionFiltersInternal({ templateId }: { templateId?: string }): JSX.Element {
+    const options = templateId?.includes('error-tracking')
+        ? FILTER_OPTIONS.filter((o) => o.value.includes('$error_tracking'))
+        : FILTER_OPTIONS
```

Comment:
Really don't like this "hack" but it gets the job done while the API for `internal_destinations` is fleshed out a little bit more